src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -0,0 +1,47 @@
+.cstg-domains-management {
```

Comment:
I'm wondering if we could potentially get rid of the black border when certain components are in focus. In this video I am using `TAB` and `SHIFT+TAB` to hop around
https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/053afe50-2f9e-4548-a444-8671c1bd33b6

Reply:
Good catch, I've removed the outline now